core/include/drivers/tpm.h

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: BSD-2-Clause
```

Comment:
I'm wondering if `core{,/include}/drivers` are good places. `<drivers/tpm.h>` sounds like it should be a driver to access a TPM chip, this is not really a driver right? Maybe this can go in `core{,include}/kernel`?

Reply:
+1 for `core{,include}/kernel`?